bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerRecoveryOp.java

```diff
@@ -110,7 +116,7 @@ private void doRecoveryRead() {
         if (!callbackDone.get()) {
             entryToRead++;
             try {
-                new RecoveryReadOp(lh, lh.bk.scheduler, entryToRead, entryToRead, this, null).initiate();
+                new RecoveryReadOp(lh, lh.bk.scheduler, entryToRead, entryToRead, this, null).parallelRead(parallelRead).initiate();
```

Comment:
@jvrao : this parallel read flag is controlled by the setting "bk.getConf().getEnableParallelRecoveryRead()". currently it is for recovery only